lib/f56/f56-dispenser.js

```diff
@@ -61,6 +61,7 @@ BillDispenser.prototype.billsPresent = function billsPresent () {
 
 BillDispenser.prototype.waitForBillsRemoved = function waitForBillsRemoved () {
   return new Promise((resolve, reject) => {
+    var retries = 3
```

Comment:
I think using var is disallowed by the linter, but this is lamassu-machine so, not sure.

```suggestion
    let retries = 3
```

Reply:
Hmm, I just checked and there's let/var in multiple files. Do you think it's worth changing?

Reply:
nah